test/test_status.py

```diff
@@ -32,3 +36,49 @@ def test_status(dirtyrepo):
     for filepath, status in git_status.items():
         assert filepath in git_status
         assert status == git_status[filepath]
+
+
+def test_status_untracked_no(dirtyrepo):
+    git_status = dirtyrepo.status(untracked_files="no")
+    not any(status & pygit2.GIT_STATUS_WT_NEW for status in git_status.values())
+
+
+@pytest.mark.parametrize(
+    "untracked_files,expected",
+    [
+        ("no", set()),
+        (
+            "normal",
+            {
+                os.path.join("untracked_dir", ""),
```

Comment:
Use `pathlib.Path` instead of `os.path`
Same in other places.

Reply:
Actually this probably must be different, since this test is failing on Windows (AppVeyor) because of different path separator.
Probably it must simply be `untracked_dir/`, since Git returns Unix-style path separators.